tests/std/tests/P1502R1_standard_library_header_units/test.cpp

```diff
@@ -0,0 +1,973 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+// N4861 [headers]/4:
+// "The headers listed in Table 21 [...] are collectively known as the importable C++ library headers.
+// [Note: Importable C++ library headers can be imported as module units (10.3). - end note]"
+// [tab:headers.cpp]: "Table 21: C++ library headers"
+
+// This EXCLUDES the <cmeow> headers in:
+// [tab:headers.cpp.c]: "Table 22: C++ headers for C library facilities"
+
+// clang-format off
+import <algorithm>;
+import <any>;
+import <array>;
+import <atomic>;
+import <barrier>;
+import <bit>;
+import <bitset>;
+import <charconv>;
+import <chrono>;
+import <codecvt>;
+import <compare>;
+import <complex>;
+import <concepts>;
+import <condition_variable>;
+import <coroutine>;
+import <deque>;
+import <exception>;
+import <execution>;
+import <filesystem>;
+// import <format>;
+import <forward_list>;
+import <fstream>;
+import <functional>;
+import <future>;
+import <initializer_list>;
+import <iomanip>;
+import <ios>;
+import <iosfwd>;
+import <iostream>;
+import <istream>;
+import <iterator>;
+import <latch>;
+import <limits>;
+import <list>;
+import <locale>;
+import <map>;
+import <memory_resource>;
+import <memory>;

```

Comment:
Yep - clang-format agrees for `#include`. I am not sure why VSCode sorting chooses the opposite order, since `'>'` (0x3E) is also less than `'_'` (0x5F).